### ISSUE TITLE
Chat-Events not working after chat triggers are entered once.

### ISSUE THREAD

**Issue by whoshenry** [FlyingPikachu/Quests#98]
I am creating a Quest where the player has to answer a load of questions and they are able to skip the question by typing 'skip' and the Chat-Event involves them advancing to the next stage and the chat trigger is obviously 'skip'. But after entering 'skip' once, and if a player wants to skip again, typing in 'skip' doesn't trigger the chat-event. Here's an example of the config I have: http://pastebin.com/NPsRaiAc

Basically, it only allows me to skip once and I'm unable to skip 2 in a row. If I answer a question correctly after skipping and then skip the question after that, then it works fine. But I can't skip 2 in a row.

**Comment by FlyingPikachu** [comment]
Please post the 'SkipQuestion1' event.

**Comment by whoshenry** [comment]
http://pastebin.com/usjvZVW2

**Comment by FlyingPikachu** [comment]
I see no-one was able to resolve your issue. If this is still a problem, please create a new ticket. Thanks!
